backends/apple/coreml/partition/coreml_partitioner.py

```diff
@@ -80,6 +81,8 @@ def partition(self, exported_program: ExportedProgram) -> PartitionResult:
                 node.meta["delegation_tag"] = tag
                 partition_tags[tag] = self.delegation_spec
 
+        tag_constant_data(exported_program)
```

Comment:
Trying to understandt:
1. So for now we just tag all constant data as supported by backend, right?
2. If we want to selectively tag consts, we will use something like `is_param(edge_exported_program, node) or is_buffer(edge_exported_program, node)` then tag ourselves?

Reply:
1. Yea, to maintain functionality, we tag_constant_data tags all the constant data which is used by partitions. This maintains the functionality that we currently have.
2. Yea if you want to selectively tag consts you can filter it however you like, whether using is_param or is_buffer, and add the tag. The call `tag_constant_data`, tags all the constant data which are used by the partitions. This maintains the functionality as before. If you don't want to handle owning and serializing the data, you can choose to not tag the data node, and instead the data node will be fed as an input to the delegate.